ui/src/api/hardware-object.js

```diff
@@ -17,7 +17,7 @@ export function sendSetValue(objectId, objectType, value) {
 }
 
 export function sendGetValue(objectId, objectType) {
-  return sendExecuteCommand(objectType, objectId, 'get_value', {});
+  return sendGetAttribute(objectType, objectId, 'get_value');
```

Comment:
A suggestion for `ui/src/api/hardware-object.js`:

```js
// It's a GET method, so the prefix should have been changed to `fetch`
export function fetchAttribute(objectType, objectId, attributeName) {
  return endpoint.get(`/${objectType}/${objectId}/${attributeName}`).safeJson();
}

// Very thin abstraction to help with readability
export function fetchValue(objectType, objectId) {
  return fetchAttribute(objectType, objectId, 'get_value');
}
```

Reply:
Thanks, well spotted